libs/ts-rest/react-query/src/index.ts

```diff
@@ -1 +1,2 @@
 export * from './lib/react-query';
+export * from './lib/types';
```

Comment:
are we sure we want to expose _all_ of the types?

Reply:
Inside the `types.ts` file, only the types that are needed are exported so we should be good here 👍